style/header.scss

```diff
@@ -57,7 +57,7 @@
       transition: height 0.4s;
       /* bump height to get big logo (bumped up by fraction of golden ratio) */
       height: calc(26px + 1/2 * calc(1.61803 * 26px - 26px));
-      body.scrolled& {
+      body.scrolled.home .site-logo img {
```

Comment:
I left a comment in the generated .css file but flagging this as the source of the issue. 

See https://sass-lang.com/playground/#eJwzNNTLyM9NVajmUlDQK84sSdXNyU/PB3MVFDJz06EsBYWk/JRKveLkovycnNQUBTW4OFAmMTk7vSi/NC/FSiEppzTVGipTywUja7lqFQAHBRvf 

I think you just want a space and then the ampersand.

Reply:
So the ampersand is what was there already, but ampersand at the end isn't supported by the version of SASS that we are using so I just replaced it with the parent.

But you're right, the output CSS doesn't seem right 🤔

Reply:
Oh wait I see the issue!

So when you include an ampersand it raises the rule up a level, if that makes sense. 

So this
```scss
.foo {
    font-weight: bold;
    .bar {
        color: red;
        
        .baz {
            font-size: 3rem;
        }
    }
}
```

becomes this

```css
.foo {
  font-weight: bold;
}
.foo .bar {
  color: red;
}
.foo .bar .baz {
  font-size: 3rem;
}
```

But this
```scss
.foo {
    font-weight: bold;
    .bar {
        color: red;
        
        &.baz {
            font-size: 3rem;
        }
    }
}
```

becomes this:
```css
.foo {
  font-weight: bold;
}
.foo .bar {
  color: red;
}
.foo .bar.baz {
  font-size: 3rem;
}
```

*not* this:

```css
.foo {
  font-weight: bold;
}
.foo .bar {
  color: red;
}
.foo .bar .foo .bar.baz {
  font-size: 3rem;
}
```

Reply:
@ashleytowner also what we had before had NO space before the ampersand, and when I tried in the playground I got something working by adding a space.

If we need to get a newer version of sass I'm fine with that too